CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+## 0.9.3
+
+### Important Changes 
+- [jdbc-v2] SQL parser from v1 is ported to v2 to address multiple issues with SQL parsing. The new parser is 
+still an option and will be developed further. To use it set `com.clickhouse.jdbc.DriverProperties#SQL_PARSER` to `ANTLR4`.
+  (https://github.com/ClickHouse/clickhouse-java/pull/2579). This fixes issue:
+  - https://github.com/ClickHouse/clickhouse-java/issues/2574
+  - https://github.com/ClickHouse/clickhouse-java/issues/2568
+  - https://github.com/ClickHouse/clickhouse-java/issues/2537
+  - https://github.com/ClickHouse/clickhouse-java/issues/2595
+  - https://github.com/ClickHouse/clickhouse-java/issues/2617
+  - https://github.com/ClickHouse/clickhouse-java/issues/2569
+  - https://github.com/ClickHouse/clickhouse-java/issues/2570
+  - https://github.com/ClickHouse/clickhouse-java/issues/2571
+  - https://github.com/ClickHouse/clickhouse-java/issues/2572
+  - https://github.com/ClickHouse/clickhouse-java/issues/2573
+  - https://github.com/ClickHouse/clickhouse-java/issues/2609
+
+- [repo] New artifact `clickhouse-jdbc-all` added to address issue when maven package qualifiers may not be used. 
+This artifact should is a copy of `clickhouse-jdbc:all` but should be used only when required. (https://github.com/ClickHouse/clickhouse-java/issues/2625) 
```

Comment:
> This artifact should is a copy
I think the should is extra here

> but should be used only when required.
We should explain in what scenarios this may be required